tests/tests_pytorch/profilers/test_profiler.py

```diff
@@ -471,24 +471,6 @@ def look_for_trace(trace_dir):
     assert look_for_trace(tmpdir / "lightning_logs" / "version_0")
 
 
-# Todo: this test has not been running as all our CI GPU runners have higher capacity
```

Comment:
So we would never take it back?

Reply:
No. The test is not great anyways. Eventually device capability 8 will be ancient